src/client/provider.ts

```diff
@@ -0,0 +1,65 @@
+/**
+ * @module Provider
+ */
+
+/**
+ * Need a comment block here so that the comment block above is interpreted as a file comment, and not a comment on the
+ * import below.
+ *
+ * @hidden
+ */
+import semverCompare from 'semver-compare';
+
+import {withStrictTimeout} from '../provider/common/async';
+
+import {tryServiceDispatch} from './connection';
+import {APITopic, ProviderStatus} from './internal';
+
+/**
+ * Retrieves the status of the Service Provider.
+ *
+ * If the Provider is connected, you will receive the Providers version number. If not, this will be null.
```

Comment:
Can we re-word? Sounds as if this function just returns the version number, rather than an object.

Reply:
✅